crates/storage/db/src/abstraction/cursor.rs

```diff
@@ -122,6 +122,66 @@ impl<'cursor, 'tx, T: Table, CURSOR: DbCursorRO<'tx, T>> std::iter::Iterator
     }
 }
 
+impl<'cursor, 'tx, T: Table, CURSOR: DbCursorRO<'tx, T>> Walker<'cursor, 'tx, T, CURSOR> {
+    /// construct Walker
+    pub fn new(
+        cursor: &'cursor mut CURSOR,
+        start: IterPairResult<T>,
+        _tx_phantom: PhantomData<&'tx T>,
```

Comment:
Can't you omit this param and have it be generated inside the function?

Reply:
fixed, much cleaner